### PR TITLE
Import meta generated cuds and keywords

### PR DESCRIPTION
**Pull request by mehdisadeghi**
@roigcarlo @kitchoi @SGGgarcia @tuopuu 

Please review this initial PR and go through the failed tests and see how we can fix them.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,2 +1,3 @@
 [flake8]
-exclude = doc/*
+# Exclude docs and generated files.
+exclude = doc/*,simphony/cuds/meta/,simphony/core/keywords.py,simphony/core/cuba.py
```
Comment (itziakos) [inline setup.cfg]: There is not real reason why the generated python code is not pep8 compatible. This should be fixed.

**Comment by kitchoi** [comment]
@mehdisadeghi @itziakos Good to merge?

**Comment by kitchoi** [comment]
The generated code decreases coverage by a lot!

**Comment by kitchoi** [comment]
Coverage could be enhanced if we bring in the tests in `simphony-metadata`

**Comment by mehdisadeghi** [comment]
Thanks for all the fixes @kitchoi ! I also saw your other branch, I'll regenerate.

**Comment by mehdisadeghi** [comment]
@kitchoi @itziakos Any idea why does the appveyor test fail?

**Comment by kitchoi** [comment]
@mehdisadeghi I believe it has to do with a new version flake8, here is the relevant issue reported on flake8: https://gitlab.com/pycqa/flake8/issues/179

**Comment by mehdisadeghi** [comment]
@kitchoi This is a symptom of the problem that I had described [before](https://github.com/simphony/simphony-framework/issues/15).

**Comment by kitchoi** [comment]
@mehdisadeghi In my last commit, I skipped the known problematic versions (3.0.0, 3.0.1) of flake8.  If new flake8 version is made available, e.g. 3.0.2, pip would still install the latest version.  It is possible that the known issue could be fixed on 3.0.2, and it would be better that we install it.  If 3.0.2 comes up and the bug is not fixed, we skip it again.

**Comment by mehdisadeghi** [comment]
@kitchoi Even though the version of pip on appveyor is the same as the one on my local machine (8.1.2), surprisingly, it does not recognize the exclusion operator that you've added:

```
50 Invalid requirement: 'flake8=3.0.1'
51 = is not a valid operator. Did you mean == ?
```

**Comment by mehdisadeghi** [comment]
I think we can proceed with merging this PR.

**Pull request merged**